datasets/fr/amf_regulatory_sanctions/fr_amf_regulatory_sanctions.yaml

```diff
@@ -0,0 +1,938 @@
+          - "et Mlle B"
+          - "*CAPITAL"
+          - "Société GROUPE X"
+          - "et sociétés X"
+          - "(anciennement dénommée Y)"
+          - "et de , anciennement dénommée X'"
+          - ", cogérants de"
+          - "C D E F"
+          - "Y"
+          - "ET Z"
+          - "et Mmes B"
+          - "(aujourd'hui dénommée X')"
+          - "(venant aux droits et obligations de )"
+          - ", MME B"
+          - "ainsi que"
+          - "(anciennement X')"
+          - "(ex ') venant aux droits de"
+          - "(venant aux droits et obligations de )"
+          - "venant au droits et obligations de"
+        entities: []
+      - match: MM. Gérard Monnet, Michael Aubourg, Davide Blei, Cyril de Bournet et Marco Perelli-Cippo
+        entities: 
+          - Gérard Monnet
+          - Michael Aubourg
+          - Davide Blei
+          - Cyril de Bournet
+          - Marco Perelli-Cippo
+      - match: La société Traditia<br /> Monsieur Philippe Le Gouz de Saint Seine
+        entities:
+          - Traditia
+          - Philippe Le Gouz de Saint Seine
+      - match: LA CAISSE D'EPARGNE ET DE PREVOYANCE ILE DE FRANCE, LA CAISSE D'EPARGNE ET DE PREVOYANCE NORMANDIE, LA CAISSE D'EPARGNE ET DE PREVOYANCE PROVENCE ALPES CORSE, LA CAISSE D'EPARGNE ET DE PREVOYANCE LOIRE DROME ARDECHE, ET LA SOCIETE X VENANT AUX DROITS DE LA SOCIETE ECUREUIL GESTION
+        entities:
+          - Caisse d'Epargne et de Prévoyance Ile de France
+          - Caisse d'Epargne et de Prévoyance Normandie
+          - Caisse d'Epargne et de Prévoyance Provence Alpes Corse
+          - Caisse d'Epargne et de Prévoyance Loire Drôme Ardèche
+          - Ecureuil Gestion
+      - match: 'venant aux droits de la société Néville Gestion'
+        entities:
+          - Neville Gestion
+      - match: 'CAISSE NATIONALE DES CAISSES D’EPARGNE ET DE PREVOYANCE, CAISSES D’EPARGNE DE PROVENCE ALPES CORSE, BOURGOGNE FRANCHE-COMTE, ILE DE FRANCE (venant aux droits des CAISSES D’EPARGNE D’ILE DE FRANCE PARIS ’ILE DE FRANCE NORD), AQUITAINE POITOU-CHARENTES (venant aux droits des CAISSES D’EPARGNE D’AQUITAINE NORD et de POITOU-CHARENTES), NORMANDIE (venant aux droits de la CAISSE D’EPARGNE DE BASSE NORMANDIE), COTE D’AZUR, ET RHONE-ALPES (venant aux droits des CAISSES D’EPARGNE DE RHONE-ALPES LYON et DES ALPES)'
+        entities:
+          - Caisse Nationale des Caisses d'Epargne et de Prévoyance
+          - Caisse d'Epargne de Provence Alpes Corse
+          - Bourgogne Franche-Comté
+          - Ile de France
+          - Caisses d'Epargne d'Ile de France Paris 'Ile de France Nord
+          - Aquitaine Poitou-Charentes
+          - Caisses d'Epargne d'Aquitaine Nord
+          - Poitou-Charentes
+          - Normandie
+          - Caisse d'Epargne de Basse Normandie
+          - Cote d'Azur
+          - Rhone-Alpes
+          - Caisses d'Epargne de Rhone-Alpes Lyon
+          - des Alpes
```

Comment:
so this is where we can really utilise the fact that we're manually untangling the names so that we don't say that the entire _administrative region_ of Bourgogne Franche-Comte has been found guilty, but only its savings bank. Same for the others.

The intuition here is that these look like place names, and having a region in france listed on our website as subject to "regulatory action" is awkward and ultimately not accurate.

```suggestion
      - match: 'CAISSE NATIONALE DES CAISSES D’EPARGNE ET DE PREVOYANCE, CAISSES D’EPARGNE DE PROVENCE ALPES CORSE, BOURGOGNE FRANCHE-COMTE, ILE DE FRANCE (venant aux droits des CAISSES D’EPARGNE D’ILE DE FRANCE PARIS ’ILE DE FRANCE NORD), AQUITAINE POITOU-CHARENTES (venant aux droits des CAISSES D’EPARGNE D’AQUITAINE NORD et de POITOU-CHARENTES), NORMANDIE (venant aux droits de la CAISSE D’EPARGNE DE BASSE NORMANDIE), COTE D’AZUR, ET RHONE-ALPES (venant aux droits des CAISSES D’EPARGNE DE RHONE-ALPES LYON et DES ALPES)'
        entities:
          - Caisse Nationale des Caisses d'Epargne et de Prévoyance
          - Caisse d'Epargne de Provence Alpes Corse
          - Caisse d'Epargne de Provence Bourgogne Franche-Comté
          - Ile de France
          - Caisses d'Epargne d'Ile de France Paris 'Ile de France Nord
          - Aquitaine Poitou-Charentes
          - Caisses d'Epargne d'Aquitaine Nord
          - Poitou-Charentes
          - Normandie
          - Caisse d'Epargne de Basse Normandie
          - Cote d'Azur
          - Rhone-Alpes
          - Caisses d'Epargne de Rhone-Alpes Lyon
          - des Alpes
```

This looks like the orrect list

![Screenshot 2024-08-13 at 12 12 55](https://github.com/user-attachments/assets/9c2a07e9-ebe0-4923-8648-43882546b2d2)